app/views/lead_providers/confirm_schools/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :title, "Confirm schools" %>
+<% content_for :before_content, govuk_back_link(text: 'Back', href: dashboard_path) %>
+
+<h1 class="govuk-heading-xl">Confirm recruitment of <%= @form.school_ids.count %> <%= "school".pluralize(@form.school_ids.count) %>, working with <%= @delivery_partner.name %>.</h1>
+
+<%= govuk_warning text: "Only report schools that you have signed a contract with." %>
+
+<table class="govuk-table">
+  <thead>
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">School</th>
+      <th scope="col" class="govuk-table__header">URN</th>
+      <th scope="col" class="govuk-table__header">Local Authority</th>
+      <th scope="col" class="govuk-table__header">Delivery partner</th>
+      <th scope="col"></th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @schools.each do |school| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><strong><%= school.name %></strong></td>
+        <td class="govuk-table__cell"><%= school.urn %></td>
+        <td class="govuk-table__cell"><%= school.local_authority&.name %></td>
```

Comment:
Uhh, N+1 queries here we come :( 
Not easily fixable either. 

@ethanmills do we have a tech debt ticket for looking into it? That's not the first place I see it in I think.

Reply:
It should be quite straight forward to fix though :)

Reply:
```
class School < ApplicationRecord
...
  def local_authority
    school_local_authorities.latest.first&.local_authority
  end
...
end
```
This doesn't look straightforward to preload to me :P But then again, I haven't spent too long in Rails query world - feel free to give it a go :)

Reply:
@kicferk1 Ugh, I see! Yeah, that is not really straight forward! :P